src/Validators/IntegrityChecks/Form/TemplatedEmailActionCheck.cs

```diff
@@ -31,6 +31,9 @@ public IntegrityCheckResult Validate(FormSchema schema)
 
                 if (string.IsNullOrEmpty(action.Properties.To))
                     result.AddFailureMessage("Templated Email Action, there is no 'To' provided");
+
+                if (action.Properties.Personlisation is null || action.Properties.Personlisation.Count.Equals(0))
+                    result.AddFailureMessage("Templated Email Action, there is no 'Personlisation' field(s) added");
```

Comment:
Will there ever be occasions where no personalisation is required for a form?

Reply:
it mainly to make sure the json is valid before proceeding with the form

Reply:
we dont want to enforce personalisation, it should not be required, this should work if i create a templated email with personalisation and without.

Reply:
I will remove it